cancel-in-progress/child_workflow.go

```diff
@@ -0,0 +1,19 @@
+package cancel_in_progress
+
+import (
+	"go.temporal.io/sdk/workflow"
+	"time"
+)
+
+// SampleChildWorkflow is a Workflow Definition
+func SampleChildWorkflow(ctx workflow.Context, name string) (string, error) {
+	logger := workflow.GetLogger(ctx)
+
+	// Simulate some long running processing.
+	_ = workflow.Sleep(ctx, time.Second*3)
```

Comment:
If you swallow this error this cannot get cancelled. It is important for our samples to show best practices of always responding to errors (if we haven't in other samples we should fix that)